### PR TITLE
Bump PHC and version to beta 2

### PR DESCRIPTION
**Pull request by tonidero**
This PR updates PHC to the latest version to get the latest fixes and bumps the version to `6.0.0-beta.2`

### CODE REVIEW

File: VERSIONS.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
 | Version      | iOS version | Android version | Common files version |
 |--------------|-------------|-----------------|----------------------|
 | 6.0.0-beta.1 | 4.25.7      | 6.9.3           | 6.1.0                |
+| 6.0.0-beta.2 | 4.25.9      | 6.9.4           | 6.1.2                |
```
Comment (NachoSoto) [inline VERSIONS.md:4]: I think newest one goes on top?

File: example/purchase-tester/src/components/FunctionTesterContainer.tsx [hunk 2]
```diff
@@ -2,12 +2,13 @@ import './FunctionTesterContainer.css';
 import {IonButton, IonCard, IonCardContent, IonCardHeader, IonCardSubtitle, IonCardTitle} from "@ionic/react";
 import React, {useEffect, useState} from "react";
 import {
+  LOG_LEVEL,
+  PRODUCT_CATEGORY,
```
Comment (vegaro) [inline example/purchase-tester/src/components/FunctionTesterContainer.tsx:6]: great

File: src/web.ts [hunk 3]
```diff
@@ -16,8 +16,8 @@ import type {
   PurchasesPromotionalOffer,
   PurchasesStoreProduct,
   ShouldPurchasePromoProductListener,
-} from '@revenuecat/purchases-typescript-internal';
-import { REFUND_REQUEST_STATUS } from '@revenuecat/purchases-typescript-internal';
+} from '@revenuecat/purchases-typescript-internal-esm';
+import { REFUND_REQUEST_STATUS } from '@revenuecat/purchases-typescript-internal-esm';
```
Comment (vegaro) [inline src/web.ts:20]: can we move this to the previous import?

**Review by vegaro — APPROVED** [review]

File: src/web.ts (hunk 3 above)
Comment (tonidero) [reply src/web.ts:20]: Oh for some reason I didn't see these comments in github before merging :/. Will fix this though 👍

**Comment by danny-hunt** [comment]
@tonidero just a heads-up that the change here to the esm version of the typescript internal package meant that I needed to add "@revenuecat/purchases-capacitor" and "@revenuecat/purchases-typescript-internal-esm" to transpilePackages in my next.config.js 

Here is my yarn.lock change
image

Which caused this error when doing `next build`:
image

Probably not a huge deal and would only affect nextjs users (I think) so feel free to ignore. I thought I'd mention it in case you want to include something in the README or otherwise. 🙂

**Comment by tonidero** [comment]
Hey @danny-hunt, thanks for reporting that. Yeah, we switched to the esm version of that library since we are using esm for this plugin, so it made the most sense.

I was trying to repro your issue with nextjs in a new nextjs application but I was able to import our library without issues (I noticed the new app also uses esnext). I wasn't using `transpilePackages` though. Can you give more details of how you reproduced this issue. Some minimum reproducible steps would be very helpful.

**Comment by danny-hunt** [comment]
@tonidero Awesome that you've looked at it already! I'm quite busy at the moment but will try to send a reproduction when I can. I do see what you mean though - it didn't reproduce on a default new nextjs set-up for me either... 
Maybe there's some weird configuration in my project 😅. Will get back to you if I work it out!

**Pull request merged**